core/controllers/mappings.py

```diff
@@ -68,11 +68,13 @@ def map_data_to_models(self, data: pd.DataFrame) -> list[db.Model]:
                     parent_lookup_1, parent_lookup_2 = parent_lookup
                     child_lookup_1, child_lookup_2 = child_lookup_column
                     lookups = {
-                        parent_lookup_1: self.get_row_id(ents.Programme, {parent_lookup_1: row[child_lookup_1]}),
-                        parent_lookup_2: self.get_row_id(ents.Submission, {parent_lookup_2: row[child_lookup_2]}),
+                        parent_lookup_1: self.get_row_id(ents.Programme, {parent_lookup_1: row.get(child_lookup_1)}),
```

Comment:
I know it's not strictly speaking part of this PR, but could we rename anything post-foxed with a _1 or _2 to be more meaningful? I have literally no idea from this code alone what they are :)